setup.cfg

```diff
@@ -17,7 +17,7 @@ minversion = 3.0
 testpaths = "sunpy" "docs"
 norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "astropy_helpers"
 doctest_plus = enabled
-addopts = -p no:warnings --doctest-rst
+addopts = --doctest-rst
```

Comment:
It took me ages to work out I had to get rid of this to show warnings whilst testing, so I have got rid of it here too to save other contributors the hassle when they are trying to use pytest.

Reply:
:thinking: I see your point, on the other hand I don't really want it making our CI build output 70 miles long because of all the warnings.

I would welcome a pr to remove most of the warnings?! :laughing:

Reply:
I'm working on the mapbase warnings at the moment, which I'll put in another PR. I generally think that tests shouldn't raise warnings, or if you expect them to raise warnings they should be caught and checked that they are the expected warnings (instead of just pretending they don't exist).

Reply:
I agree that we should do it, I just don't think we should do it here.

Reply:
Yep, agreed.